### ISSUE TITLE
Bot always shows APR: Bid

### ISSUE THREAD

**Issue by b-seite** [ctubio/Krypto-trading-bot#274]
Hey Charles,

i recognized that the bots always showing up APR:bid don't matter what i change in the settings. even when i switch to apMode manual and pdiv 100%
don't know if its just a GUI problem so i switched off apr for the moment

and btw. (maybe its another issue) when i switch to appMode to manual, the next recalculation of the TBP is when a trade was made. Could make Problems in some Situations i guess.

Thanks Boris
---
```
app exchange: Coinbase/BTC/EUR
app version: undisclosed
```
![300px-spock_vulcan-salute3](https://cloud.githubusercontent.com/assets/1634027/22077151/4110e73e-ddb3-11e6-9d84-358e9f133d34.png)

**Comment by ctubio** [comment]
Hii'', you mean TBP should be calculated right after you save the new quoting parameters? (but currently is calculated only after the first trade?)

**Comment by b-seite** [comment]
Yes exactly
